### ISSUE TITLE
Officially support Chromium's new documentPictureInPicture API

### ISSUE THREAD

**Issue by jpodwys** [twilio/twilio-video.js#2044]
Twilio employee Donal Toomey requested that I create this issue.

I'd like to request official [documentPictureInPicture](https://developer.chrome.com/docs/web-platform/document-picture-in-picture) support. Today, Twilio's very nice efficiency features (pausing tracks when a tab is backgrounded) result in pausing all video feeds in a documentPictureInPicture context. I've forked Twilio's SDK to support this. Would love to see official support so we can get back onto the main repo.

I'm also happy to share the changes I made, although they were more brute force than surgical.

Thank you!

**Comment by jpodwys** [comment]
@donaltoomey is there any update here?

**Comment by charliesantos** [comment]
@jpodwys we have a work in progress PR for this feature. Can you please have a look and see if it addresses your needs? TIA! https://github.com/twilio/twilio-video.js/pull/2093/files

**Comment by jpodwys** [comment]
Thank you for working on this!
I'll need to get work to approve some time for me to pull this branch, build it, and test it locally.

**Comment by jpodwys** [comment]
Sorry for the delay - I was able to get this prioritized. Unfortunately, the latest release (2.31.0) still permanently pauses video tracks after backgrounding the browser tab that spawned the PiP window. Here's a screen recording showing this behavior. Sorry for the low resolution and fast talking - github's filesize restriction necessitated it. I'm happy to share more info and/or a higher resolution version of the video directly if it would be helpful.

https://github.com/user-attachments/assets/0c69d9e2-c782-483a-80a7-cf84b8eaa951

**Comment by charliesantos** [comment]
Thank you for trying it out @jpodwys . Can you provide me access to the app that reproduces the issue? Also, are you able to reproduce it using our quickstart? https://github.com/twilio/video-quickstart-js/

**Comment by jpodwys** [comment]
Yes, I'm able to reproduce this using Twilio's quickstart application. Here's a screen recording

https://github.com/user-attachments/assets/3283601e-05ad-4305-b9f4-86c9825e536e

**Comment by charliesantos** [comment]
Thanks @jpodwys . We'll take a look at it more.

**Comment by jpodwys** [comment]
Thank you!

**Comment by jpodwys** [comment]
Thank you! After a quick test, 2.32.1 does appear to fix the issue.
